playbooks/byo/openshift-checks/health.yml

```diff
@@ -1,3 +1,10 @@
 ---
 - include: ../openshift-cluster/initialize_groups.yml
+  tags:
+  - always
```

Comment:
Running `ansible-playbook ... --tags=foo` fails with our plays/roles. It's never tested in CI. I doubt there will ever be a use case for it except perhaps in development with playbooks of very limited scope.

There may be some value in running `ansible-playbook ... --skip-tags=foo`; however in that case `tags: always` still seems unnecessary. I too would like to know if there's some reason to keep this that I'm not seeing.